creator-node/src/routes/nodeSync.js

```diff
@@ -199,6 +200,14 @@ async function _nodesync (req, walletPublicKeys, creatorNodeEndpoint) {
         throw new Error(`Malformed response received from ${creatorNodeEndpoint}. "walletPublicKey" property not found on CNodeUser in response object`)
       }
       const fetchedWalletPublicKey = fetchedCNodeUser.walletPublicKey
+      const myCnodeEndpoint = await middlewares.getOwnEndpoint(req)
+      const userReplicaSet = await middlewares.getCreatorNodeEndpoints(req, fetchedWalletPublicKey)
+
+      // gateways to try  is user metadata + replica set (excluding self). Spread + set uniq's the array
+      const gatewaysToTry = [...new Set([
+        config.get('userMetadataNodeUrl'),
```

Comment:
already discussed but making a note here - need to handle case where this val is null/empty